src/templates/zh-CN/2019/methodology.html

```diff
@@ -356,10 +356,9 @@ <h3 id="rework-css"><a href="#rework-css" class="anchor-link">Rework CSS</a></h3
       </p>
 
       <p>
-        在<a href="{{ url_for('chapter', year=year, lang=lang, chapter='css') }}">CSS</a> 章节，这个特殊用途的工具显著地提高了数据库中许多度量标准的准确性。所有外部样式表中的CSS和每个页面的内联样式块都被解析和查询，让这个分析工作成为可能。阅读<a href="https://discuss.httparchive.org/t/analyzing-stylesheets-with-a-js-based-parser/1683">这段</a> 来了解更多关于它如何同BigQuery上的HTTP Archive数据集来集成.
+        在<a href="{{ url_for('chapter', year=year, lang=lang, chapter='css') }}">CSS</a> 章节，这个特殊用途的工具显著地提高了数据库中许多度量标准的准确性。所有外部样式表中的CSS和每个页面的内联样式块都被解析和查询，让这个分析工作成为可能。阅读<a href="https://discuss.httparchive.org/t/analyzing-stylesheets-with-a-js-based-parser/1683">这段</a> 来了解更多关于它如何同BigQuery上的HTTP Archive数据集来集成。
       </p>
     </section>
-
```

Comment:
Can we add back this deleted line so the line numbers stay in sync with English version?